source/extensions/compression/gzip/decompressor/zlib_decompressor_impl.h

```diff
@@ -49,7 +69,19 @@ class ZlibDecompressorImpl : public Zlib::Base,
   int decompression_error_{0};
 
 private:
+  // TODO: clean up friend class. This is here to allow coverage of chargeErrorStats as it isn't
```

Comment:
Nit: you may want to have `TODO(junr03)` in this line.

Reply:
I like leaving todos I don't intend to do right away without a username that way it doesn't dissuade others to fix todos they find.